go.mod

```diff
@@ -11,7 +11,8 @@ require (
 	github.com/prometheus/client_golang v1.0.0
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
+	go.uber.org/automaxprocs
```

Comment:
should we set the version of automaxprocs?

Reply:
done